README.md

```diff
@@ -22,6 +22,98 @@ The module has been tested with:
 - PostgreSQL 15
 - terraform-helm-materialize v0.1.12 (Materialize Operator v25.1.7)
 
+## Disk Support for Materialize on GCP
+
+This module supports configuring disk support for Materialize using local SSDs in GCP with OpenEBS and lgalloc.
+
+### Machine Types with Local SSDs in GCP
+
+When using disk support for Materialize on GCP, you need to use machine types that support local SSD attachment. Here are some recommended machine types:
+
+1. [N2 series](https://cloud.google.com/compute/docs/general-purpose-machines#n2d_machine_types) with local SSDs:
+   - For memory-optimized workloads similar to AWS r7gd, consider `n2-highmem-16` or `n2-highmem-32` with local SSDs
+   - Example: `n2-highmem-32` with 2 or more local SSDs
+
+2. [C2 series](https://cloud.google.com/compute/docs/compute-optimized-machines#c2_machine_types) with local SSDs:
+   - For compute-optimized workloads
+   - Example: `c2-standard-16` with local SSDs
```

Comment:
Please remove the compute optimized types. 99% of our customers will be memory limited.